README.md

```diff
@@ -561,10 +562,18 @@ Provided there is a new line between the opening and closing braces / tag, it ca
 
 Based on [vim-sneak](https://github.com/justinmk/vim-sneak), it allows for jumping to any location specified by two characters.
 
-| Setting                            | Description                                                 | Type    | Default Value |
-| ---------------------------------- | ----------------------------------------------------------- | ------- | ------------- |
-| vim.sneak                          | Enable/disable vim-sneak                                    | Boolean | false         |
-| vim.sneakUseIgnorecaseAndSmartcase | Respect `vim.ignorecase` and `vim.smartcase` while sneaking | Boolean | false         |
+| Setting                            | Description                                                                                                                                                                                                                | Type    | Default Value            |
+| ---------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- | ------------------------ |
+| vim.sneak                          | Enable/disable vim-sneak                                                                                                                                                                                                   | Boolean | false                    |
+| vim.sneakUseIgnorecaseAndSmartcase | Respect `vim.ignorecase` and `vim.smartcase` while sneaking                                                                                                                                                                | Boolean | false                    |
+| vim.sneakMaxLinesToConsider        | The maximum number of lines searched (starting from the position of the cursor) while sneaking (useful if one is working with large files). -1 means no restriction.                                                       | Number  | -1                       |
```

Comment:
-1 strikes me as a slightly weird sentinel value. Any reason it's not 0 or `undefined` instead?

Reply:
I agree. The user could want to search only the current line though. At the moment maxLinesToConsider === 1 would mean to search the current line and the one below (if searching forward) and maxLinesToConsider === 0 would mean to search only the current line. I will put null as a default value as it makes more sense. Let me know if changing the behavior of 0 and 1 is required too.

Reply:
Maybe most natural to me is:

- Anything negative: invalid (I think you can set a minimum value via JSON schema)
- 0: all lines, default
- 1: just the current line
- 2: current line and next (or current line and previous)

Any issue with that scheme?

Reply:
None at all, I'll make the change soon.